docs/cli.md

```diff
@@ -11,7 +11,7 @@ prettier [options] [file/dir/glob ...]
 
 :::note
 
-To run your locally installed version of Prettier, prefix the command with `npx` or `yarn` (if you use Yarn), i.e. `npx prettier --help`, or `yarn prettier --help`.
+To run your locally installed version of Prettier, prefix the command with `npx` or `yarn exec` (if you use Yarn), i.e. `npx prettier --help`, or `yarn exec prettier --help`.
```

Comment:
To run your locally installed version of Prettier, prefix the command with `npx`, `yarn exec`, `pnpm exec`, or `bun exec`, i.e. `npx prettier --help`, `yarn exec prettier --help`, `pnpm exec prettier --help`, or `bun exec prettier --help`.

Reply:
Don't forget https://github.com/prettier/prettier/blob/main/website/versioned_docs/version-stable/cli.md